### PR TITLE
Check for unique registry for ta

### PR DESCRIPTION
**Pull request by prmittal**

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 [hunk 1]
```diff
@@ -30,7 +30,7 @@ function Locate-TestVersion()
 	{
 		$regPath = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\DevDiv\vstf\Servicing"
 	}
-	$keys = Get-Item $regPath | %{$_.GetSubKeyNames()}
+	$keys = Get-ChildItem $regPath | Where-Object {$_.GetSubKeyNames() -contains "testagentcore"}
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: if path doesn't exist. It should return empty array. just check once

Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: ignore case?

**Comment by nigurr** [comment]
task.json update? Approved

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 (hunk 1 above)
Comment (prmittal) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: checked

---
In reply to: [68206532](https://github.com/Microsoft/vsts-tasks/pull/1934#discussion_r68206532) [](ancestors = 68206532)

Comment (prmittal) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: contains is case insensitive

---
In reply to: [68206914](https://github.com/Microsoft/vsts-tasks/pull/1934#discussion_r68206914) [](ancestors = 68206914)

**Comment by nigurr** [comment]
:shipit:

**Pull request merged**